### ISSUE TITLE
why printed “decode” same？ 

### ISSUE THREAD

**Issue by BrianGong** [ilovin/lstm_ctc_ocr#13]
hi， 
thanks for your code， but I run the code in TF 1.2 get the print information like this：
![image](https://user-images.githubusercontent.com/4621114/29010885-4826b368-7b61-11e7-9ca2-2d044d0df3dd.png)

after training for 8 hours ， the accuracy is 0 ,and sevral “decoded” in seq  are same. 
is that correct?

**Comment by ilovin** [comment]
No, if you cannot see a significant change in the decoded code for about epoch, you may consider adjusting parameters.
I notice that the "origin" code are ranging in 0-10, are you using your own data?
If so, how many pics are you using?

**Comment by BrianGong** [comment]
thanks for your help!!!
use more training data（128k is ok）and wait longer time ，now the accuracy close to 1.
but I run the test.sh ,cant restore model from check point folder. the error like this 
![image](https://user-images.githubusercontent.com/4621114/29062134-10e70a04-7c53-11e7-93c5-065755276771.png)
,I read the code and test train.sh modify "restore = 0 " to "resotre = 1",the code works well, 
 do you hava and idea?  I use tensorflow 1.2.

**Comment by ilovin** [comment]
I forget to change the `LSTM_test`, the network in `LSTM_test` shall be the same as `LSTM_train`

**Comment by ilovin** [comment]
If you do not have further question please close the issue
